worker-sys/src/ext/asset_handler.rs

```diff
@@ -0,0 +1,13 @@
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(module = "__STATIC_CONTENT_MANIFEST")]
+extern "C" {
+    #[wasm_bindgen(js_name = "default")]
+    static MANIFEST_STR: String;
+}
+
+pub static ASSET_MANIFEST: Lazy<HashMap<&str, &str>> =
```

Comment:
We should avoid things other than wasm-bindgen glue in the `worker-sys` crate, this global should be moved to the `worker` crate.

Reply:
I took a different approach.
It's true that all the boilerplate code can be in a different crate. 
So the essential function that is missing to anyone do that is a function that converts the filenames under `/public`:
`favicon.ico` -> `favicon.<hash>.ico` according to the value stored in the manifest file.

In the new changes, we expose this function through the `env` variable.